onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -250,6 +252,28 @@ TEST(NnapiExecutionProviderTest, TestQDQConvModel) {
       << "No nodes should have been taken by the NNAPI EP";
 }
 
+#if defined(__ANDROID__)
+TEST(NnapiExecutionProviderTest, TestQDQModel) {
+  onnxruntime::Model model("nnapi_qdq_test_graph", false, DefaultLoggingManager().DefaultLogger());
+  Graph& graph = model.MainGraph();
+  ModelTestBuilder helper(graph);
+
+  auto build_test_case = BuildTestCase<uint8_t, uint8_t, int32_t, uint8_t>({1, 12, 37} /*input_shape*/,
+                                                                           {32, 12, 5} /*weights_shape*/);
+  build_test_case(helper);
+  helper.SetGraphOutputs();
+  ASSERT_TRUE(model.MainGraph().Resolve().IsOK());
+
+  std::vector<uint8_t> model_data((std::istreambuf_iterator<char>(model)), std::istreambuf_iterator<char>());
```

Comment:
Does this actually work?

Reply:
switch back to use model_data string per suggestion which should work...